docs/docusaurus.config.js

```diff
@@ -119,6 +119,63 @@ const config = {
                             },
                         ],
                     },
+                    {
+                        title: 'How-to guides',
```

Comment:
Is the order for these links intentional? I'm curious why Angular is first but more popular platforms like iOS, React Native etc are lower.

I don't have a strong preference either way - maybe alphabetical order makes it easier to skim through the list and makes it easier to decide what goes first.

Reply:
Yes I think alphabetical here?